### DISCUSSION TITLE
About prediction of deepGP

### DISCUSSION THREAD

**Discussion by WeichenWangbit** [cornellius-gp/gpytorch#2160] [Q&A]
Hello, according to the reference article, it seems that the amount of marginal likelihood calculation in the prediction will be affected by the number of initial data sets, is it so? However, in the program, it seems that the prediction time does not increase significantly when the dataset is enlarged. How should this be understood?

**Comment by gpleiss** [comment]
The deep GP implementation uses a standard inducing point approximation, which produces predictions in O(M^3) time (where M is the number of inducing points). The prediction time does not depend on N.
